pinery-ws/src/test/java/ca/on/oicr/pinery/ws/SampleProvenanceResourceTest.java

```diff
@@ -146,11 +146,18 @@ public void testJsonArraySize() throws Exception {
    */
   @Test
   public void testV1ProvenanceTransform() {
-    VersionTransformer<SampleProvenance> transformer = SampleProvenanceResource.transformers
-        .get("v1");
+    VersionTransformer<SampleProvenance> transformer = SampleProvenanceResource.transformers.get("v1");
     SampleProvenance sp = transformer.transform(makeBaseSampleProvenance());
     // This hash must never change
-    assertEquals("54d16cc61c3bb139ac16a017fac31f088b3809f67c07ac71e77750f0daaac07d", sp.getVersion());
+    assertEquals("c06f3bd04de538704689c77a95353c5e1ba3b512e53382662267703159bc65f7", sp.getVersion());
```

Comment:
Note: This changed because of the "Pool Name" attribute I had included in the Sample attributes, which isn't a real thing. Added `LimsLaneAttribute` and `LimsSequencerRunAttribute` enums to keep these separate and prevent similar future confusion